apps/snitch_core/config/test.exs

```diff
@@ -21,3 +21,19 @@ config :logger, level: :info
 config :argon2_elixir,
   t_cost: 1,
   m_cost: 8
+
+config :snitch_core, Snitch.Tools.ElasticsearchCluster,
+  url: "http://localhost:9200",
```

Comment:
Can we have this Elasticsearch config in `config.exs`.
We have `.env` for `test` also, so we can fetch the config values from environment.

Reply:
hmm, @SagarKarwande can you please mention the pros and cons as well. Thanks

Reply:
hmm, makes sense to me. @pkrawat1 what do you think?

Reply:
I'm pushing it to config. I was facing issue while creating an index for testing.